identity-iota/Cargo.toml

```diff
@@ -12,6 +12,7 @@ homepage = "https://www.iota.org"
 
 [dependencies]
 async-trait = { version = "0.1", default-features = false }
+form_urlencoded = { version = "1.0.1" }
```

Comment:
Just specifying `1.0` should be enough; this will help prevent conflicting versions

Reply:
Done.